lib/game.rb

```diff
@@ -0,0 +1,67 @@
+require_relative './score'
+
+class Game
+  attr_reader :throws, :score, :frame_count, :last_frame
+
+  MAX_FRAME = 10
+
+  def initialize(score = Score.new)
+    @throws = []
+    @frame_complete = true
+    @score = score
+    @frame_count = 1
+    @last_frame = []
+  end
+
+  def add(pins)
+    @throws << pins
+    play_last_frame(pins) if last_frame?
+    frame_status(pins)
+    setup_next_roll(pins)
+  end
+
+  def frame_status(pins)
+    @frame_complete = !@frame_complete if pins != 10
+  end
+
+  def frame_complete?
+    @frame_complete
+  end
+
+  def setup_next_roll(pins)
+    if frame_complete?
+      @score.calculate(@throws)
+      new_frame
+    else
+      @score.update_if_needed(pins)
+    end
+  end
+
+  def new_frame
+    @throws.clear
+    @frame_count += 1
+  end
+
+  def last_frame?
```

Comment:
This method is called the same thing as the @last_frame - makes it seem like a getter method but it isn't. Perhaps one of them could be renamed?